alpine/src/test/scala/io/delta/alpine/internal/ReadOnlyLogStoreSuite.scala

```diff
@@ -16,39 +16,51 @@
 
 package io.delta.alpine.internal
 
-import java.io.File
+import java.io.{BufferedReader, File, FileNotFoundException, InputStreamReader}
+import java.nio.charset.StandardCharsets.UTF_8
+import java.util.stream.Collectors
 
 import collection.JavaConverters._
 
-import io.delta.alpine.internal.storage.ReadOnlyLogStoreProvider
-import io.delta.alpine.internal.util.ConversionUtils
-import org.apache.hadoop.fs.Path
+import io.delta.alpine.internal.storage.{HDFSReadOnlyLogStore, ReadOnlyLogStoreProvider}
+import io.delta.alpine.sources.AlpineHadoopConf
+import io.delta.alpine.ReadOnlyLogStore
+import org.apache.commons.io.IOUtils
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileStatus, Path}
 
 import org.apache.spark.sql.delta.storage.LogStore
 import org.apache.spark.sql.QueryTest
 import org.apache.spark.sql.test.SharedSparkSession
 
-abstract class ReadOnlyLogStoreSuite extends QueryTest
-  with ReadOnlyLogStoreProvider
-  with SharedSparkSession
-  with ConversionUtils {
+// TODO private[internal] ??
+abstract class ReadOnlyLogStoreSuiteBase extends QueryTest
```

Comment:
It's not worth to run these tests for so many `LogStore`s. They have the same implementation. We only need to test `HDFSReadOnlyLogStore` + a test that verifies `createLogStore` returns the correct class (the test `instantiation through hadoopConf` you added)